app/assets/javascripts/fe/fe.public.nojquery.js.erb

```diff
@@ -100,7 +120,10 @@
       if( match != null )
         {
           var page = match[1];
-          if ($('#'+page).length > 0) {
+          if (response.match(/<html/)) {
```

Comment:
No `Fe.bootstrap` check here?

Reply:
No, this is a generic error handle for response that doesn't have html in it.

Seeing this again, I'd liket o take out that "we've been notified" part as it may not necessarily be true..